### PR TITLE
Wait for iframe doesn't work for already loaded iframes (close #12)

### LINKED ISSUE #12: Wait for iframe doesn't work for already loaded iframes

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @AlexanderMoskovkin

### CODE REVIEW

**Comment by helen-dikareva** [comment]
FPR

File: src/templates/globals.mustache.js [hunk 1]
```diff
@@ -56,16 +59,23 @@ window.QUnitGlobals = {
...
 
-                    iframe.contentWindow.addEventListener('load', function () {
-                        window.clearTimeout(timeoutId);
-                        resolve();
-                    });
-                }
-            }, CHECK_PREDICATE_INTERVAL);
+            if (isIframeLoaded) {
+                window.clearTimeout(timeoutId);
+                resolve();
```
Comment (AlexanderMoskovkin) [inline src/templates/globals.mustache.js:73]: `return;` requires. We should not add a listener to the `load` event if the iframe is already loaded

File: src/templates/globals.mustache.js [hunk 2]
```diff
@@ -56,16 +59,23 @@ window.QUnitGlobals = {
                 start();
             }, WAIT_FOR_IFRAME_TIMEOUT);
 
-            intervalId = window.setInterval(function () {
-                if (iframe && iframe.contentWindow) {
-                    window.clearInterval(intervalId);
+            try {
+                isIframeLoaded = iframe.contentWindow && iframe.contentWindow.document.readyState === 'complete';
```
Comment (AlexanderMoskovkin) [inline src/templates/globals.mustache.js]: I'm not sure but I think in some browsers `iframe.contentWindow.document` is null when the iframe is just added. It this case we go to the `catch` block and set the 'isIframeLoaded' to true but it should be false.
Lets check `iframe.contentWindow && iframe.contentWindow.document && ... === 'complete'`

**Comment by AlexanderMoskovkin** [comment]
and please increase the version in the `package.json`

**Comment by AlexanderMoskovkin** [comment]
no, don't increase the version. I'll do this in a separate commit

**Comment by helen-dikareva** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**